Update submodule

jrvollmer opened the pull request.

EUdds commented on Backend/package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "chase-car-dashboard-backend",
-  "version": "1.1.2",
```
Do we have versioning guidelines? Like what defines a major or minor or minor minor change?

jrvollmer replied:
Soon to come, but for the sake of not blocking front end development, this needs to be merged first.

EUdds reviewed (changes requested):
I hit the big red buttom

jrvollmer committed:
Update submodule and metadata

Update charge_enable nominal range, as well as Solar Array subcategories
and signals.

jrvollmer committed:
Update version for data format updates
